src/store/reducers/todoReducer.ts

```diff
@@ -1,12 +1,16 @@
 import { ITask } from "../../types/task.interface";
+import { TodoActionTypes } from "../actionTypes/actionTypes";
 
 const initialState: ITask[] = [];
 
 export const todoReducer = (
   state: ITask[] = initialState,
-  action: { type: string }
+  action: { type: string; payload?: any }
```

Comment:
Why any?

Reply:
For now, as quick fix I define payload type as task type. I used any here, because in future, when we`ll be able to delete task and change complete status... payloads will be different types, and, for now, I can`t define universal type, so I use any(